bee-message/src/byte_cost.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::output::Output;
+
+use packable::PackableExt;
+use serde::Deserialize;
+
+const DEFAULT_BYTE_COST: u64 = 500;
+const DEFAULT_BYTE_COST_KEY_WEIGHT: u64 = 10;
+const DEFAULT_BYTE_COST_DATA_WEIGHT: u64 = 1;
+
+/// Builder for a [`ByteCostConfig`].
+#[derive(Default, Deserialize)]
```

Comment:
Can you put Deserialize under the `serde1` feature like other types of this crate please